GliaWidgetsTests/Mocks/CoreSDKClient.Operator.Mock.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import SalemoveSDK
+@testable import GliaWidgets
+
+extension CoreSdkClient.Operator {
+    static var mock: CoreSdkClient.Operator {
```

Comment:
I also made this in my visitor transfer PR. As noted there, it should be a function. Perhaps you should change it to function too so the merge conflicts wont be as bad?

Reply:
19842e64d7c7c6f21a03c0c2a55f601583483950